site/public/css/select-widgets.css

```diff
@@ -24,8 +24,8 @@
     color: var(--standard-dark-gray);
 }
 
-/* stylelint-disable-next-line selector-id-pattern */
 [data-theme="dark"]
-    #select2-time_zone_drop_down-container.select2-selection__rendered {
+/* stylelint-disable-next-line selector-id-pattern */
+#select2-time_zone_drop_down-container.select2-selection__rendered {
     color: var(--text-black);
 }
```

Comment:
What's going on here?  Isn't the style specifically supposed to be applied in dark mode?